WordPressAuthenticator/Services/LoginFacade.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  LoginFacade.swift
+//  WordPressAuthenticator
+//
+//  Created by Diego Rey Mendez on 12/08/2020.
+//  Copyright © 2020 Automattic. All rights reserved.
```

Comment:
Can you remove this? We don't normally leave them.

Reply:
Yeah, that shouldn't be there.